examples/bull-spread/bull-spread.go

```diff
@@ -70,59 +66,55 @@ func main() {
 	}
 
 	underlying := "INTC"
-	td, err := svc.mdClient.GetLatestTrade(underlying, marketdata.GetLatestTradeRequest{Feed: "iex"})
+	td, err := svc.mdClient.GetLatestTrade(underlying, marketdata.GetLatestTradeRequest{})
 	if err != nil {
 		log.Fatalf("getting latest trade for symbol %s: %v", underlying, err)
 	}
 	px := td.Price
 
 	// list contracts that are expiring at least a week from now and strike is GTE latest trade
-	now := time.Now()
-	dt := civil.DateOf(time.Date(now.Year(), now.Month(), now.Day()+oneWeek, 0, 0, 0, 0, time.UTC))
+	dt := civil.DateOf(time.Now()).AddDays(7)
 
 	// Select two contracts for the two legs of the spread:
 	// 1. long leg, strike A at latest trade
 	var (
 		long  alpaca.OptionContract
 		short alpaca.OptionContract
 	)
-	{
-		req := alpaca.GetOptionContractsRequest{
-			UnderlyingSymbols: underlying,
-			Status:            alpaca.OptionStatusActive,
-			ExpirationDateGTE: dt,
-			StrikePriceGTE:    decimal.NewFromFloat(px), // strike A
-			TotalLimit:        1,
-		}
-		contracts, err := svc.tdClient.GetOptionContracts(req)
-		if err != nil {
-			log.Fatalf("listing contracts: %v", err)
-		}
-		if len(contracts) < 1 {
-			log.Fatal("no contracts to choose from for the long leg")
-		}
-		long = contracts[0]
-		log.Printf("-> contract to be bought long: %s", long.Symbol)
+	req := alpaca.GetOptionContractsRequest{
+		UnderlyingSymbols: underlying,
+		Status:            alpaca.OptionStatusActive,
+		ExpirationDateGTE: dt,
+		StrikePriceGTE:    decimal.NewFromFloat(px), // strike A
+		TotalLimit:        1,
+	}
+	contracts, err := svc.tdClient.GetOptionContracts(req)
+	if err != nil {
+		log.Fatalf("listing contracts: %v", err)
+	}
+	if len(contracts) < 1 {
+		log.Fatal("no contracts to choose from for the long leg")
 	}
+	long = contracts[0]
+	log.Printf("-> contract to be bought long: %s", long.Symbol)
+
 	// 2. short leg, strike B at $10 above latest trade
-	{
-		req := alpaca.GetOptionContractsRequest{
-			UnderlyingSymbols: underlying,
-			Status:            alpaca.OptionStatusActive,
-			ExpirationDateGTE: dt,
-			StrikePriceGTE:    decimal.NewFromFloat(px + 10), // strike B
-			TotalLimit:        1,
-		}
-		contracts, err := svc.tdClient.GetOptionContracts(req)
-		if err != nil {
-			log.Fatalf("listing contracts: %v", err)
-		}
-		if len(contracts) < 1 {
-			log.Fatal("no contracts to choose from for the short leg")
-		}
-		short = contracts[0]
-		log.Printf("-> contract to be shorted: %s", short.Symbol)
+	req = alpaca.GetOptionContractsRequest{
+		UnderlyingSymbols: underlying,
+		Status:            alpaca.OptionStatusActive,
+		ExpirationDateGTE: dt,
+		TotalLimit:        1,
+	}
```

Comment:
No need for this.

Reply:
a0545df8bfa0e00b8922f8b06424d92b6fc7995a